src/ext/processObliv/processObliv.ml

```diff
@@ -594,6 +634,12 @@ let rec codegenUncondInstr (instr:instr) : instr = match instr with
         | BNot -> setUnop "__obliv_c__setBitwiseNot" v e loc
         | LNot -> setUnop "__obliv_c__setLogicalNot" v e loc
         end
+    | TFloat(kind, a) when hasOblivAttr a ->
+        begin match op with
+        | Neg -> setUnop "__obliv_c__setNegF" v e loc
+        | BNot -> E.s (E.error "Bitwise operation on float invalid!\n")
```

Comment:
Alright, I see what you mean; I'll update the error message to be more specific.